webapi/Models/Response/BotResponsePrompt.cs

```diff
@@ -9,6 +9,29 @@ namespace CopilotChat.WebApi.Models.Response;
 /// </summary>
 public class BotResponsePrompt
 {
+    /// <summary>
+    /// Information about semantic dependencies of the prompt.
+    /// </summary>
+    public class SemanticDependency<T>
```

Comment:
Context for other semantic dependencies will be added in a separate PR: https://github.com/microsoft/chat-copilot/issues/150